sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md

```diff
@@ -1,14 +1,11 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.2.0 (2023-09-01)
```

Comment:
Do we have any API changes? If not, we should release a patch version. Also, fix the release date `2023-01-09`